---
float64 i_filt_param
float64 vel_filt_param
float64 i_max_param
